.protolint.yaml

```diff
@@ -49,7 +50,7 @@ lint:
     # MAX_LINE_LENGTH rule option.
     max_line_length:
       # Enforces a maximum line length
-      max_chars: 80
+      max_chars: 150
```

Comment:
Realized that the new Protobuf formatter doesn't necessarily split lines (for example, for long RPC definitions)